cslbot/helpers/textutils.py

```diff
@@ -368,12 +368,34 @@ def gen_sanitize(msg):
 def gen_intensify(msg):
     return "[%s INTENSIFIES]" % msg.upper()
 
+def gen_djones(msg):
+    keyboard = [
+            'qwertyuiop',
+            '\0asdfghjkl',
+            '\0\0zxcvbnm\0\0',
```

Comment:
Why embedded nulls?

Reply:
Nulls make the array rectangular so vertical offsets make sense